### DISCUSSION TITLE
[FEATURE] 添加 zed（后续会支持 web）

### DISCUSSION THREAD

**Discussion by Jengro777** [1Panel-dev/1Panel#6633] [Ideas]
### 1Panel 版本

1.17.18

### 请描述您的需求或者改进建议

https://github.com/zed-industries/zed

**Comment by wanghe-fit2cloud** [comment]
我查看了项目的 README 和文档，似乎没有提供容器化部署的教程。如果确实如此，那么将无法支持该项目上架到应用商店。
